server/helpers/verifyEmailMessage.js

```diff
@@ -16,7 +24,7 @@ const verifyEmailMessage = (token) => {
     `<div style="height: 20em, background-color: #E6FFED; 
       border: 1px solid black; padding: 0.5em;">
         <p>
-          <a href="http://localhost:3001/api/test/verify?token=${token}">
+          <a href="${protocol}://${address}/api/v1/users/verify?token=${token}">
```

Comment:
You can change this to ""
```suggestion
          <a href="${process.env.API_BASE_URL}users/verify?token=${token}">
```
Then have the `http://localhost/api/v1/` in the env file.

Reply:
That way we can avoid those plenty checks above.